desktop-environment.el

```diff
@@ -164,6 +164,23 @@ portion of the screen."
   :type 'string)
 
 
+
+;;; Customization - keyboard layout
+
+(defcustom desktop-environment-keyboard-layouts '("us" "us -variant dvorak" "cz")
+  "List of layouts to toggle between with.
+
+This is prepended by \"setxkbmap -layout\".
```

Comment:
Each element is a space-separated string of options passed to "setxkbmap -layout".